dbt_subprojects/daily_spellbook/models/evaa/ton/evaa_ton_liquidate.sql

```diff
@@ -0,0 +1,111 @@
+{{ config(
+       schema = 'evaa_ton'
+       , alias = 'liquidate'
+       , materialized = 'incremental'
+       , file_format = 'delta'
+       , incremental_strategy = 'merge'
+       , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+       , unique_key = ['tx_hash']
```

Comment:
can we include more than just `tx_hash` in unique key (and also in join conditions in model)?
trino has historically had poor query planning on these field types alone.
maybe `block_date` too?

Reply:
also remember to update the unique test in schema file after plz 🙏